.github/workflows/command-dispatch.yml

```diff
@@ -14,7 +14,6 @@ env:
   PULUMI_API: https://api.pulumi-staging.io
   PULUMI_GO_DEP_ROOT: ${{ github.workspace }}/..
   PULUMI_LOCAL_NUGET: ${{ github.workspace }}/nuget
-  PULUMI_MISSING_DOCS_ERROR: true
```

Comment:
How come we're getting rid of this? I set this to `true` after resolving all missing docs.

Reply:
Some of the new resources upstream have no docs I'm afraid :/

Reply:
Let's add something like this (either the literal code or a func that returns the same) and keep this flag on so that we normalize missing docs as an error to be fixed: https://github.com/pulumi/pulumi-terraform-bridge/pull/494

Reply:
Fixed